src/types.ts

```diff
@@ -1040,3 +1041,14 @@ export interface SmartAccountSigner {
   /** Custom method for populating transaction requests. */
   transactionBuilder?: TransactionBuilder;
 }
+
+/** Contains parameters for finalizing the L2->L2 deposit transaction. */
```

Comment:
It probably would make sense to mention that pre-v26 withdrawals were special kind of transaction, but starting from v26 any cross-chain token movement is called a deposit, regardless of direction.

This would help people who are not familiar with the new system to understand it easier.

Also is it L2->L2? Shouldn't it be L2->L1?